### ISSUE TITLE
Start an ephemeral workspace fails

### ISSUE THREAD

**Issue by l0rd** [eclipse/che#21164]
### Describe the bug

I have tried setting attribute at the root of the devfile as mentioned in the DevWorkspace documentation

```
attributes:
  controller.devfile.io/storage-type: ephemeral
```

I have also tried to set “Temporary storage” on from the Dashboard.  

In both cases PVC are attached to the workspace pod:

<img width="1416" alt="image" src="https://user-images.githubusercontent.com/606959/153860496-12085de9-d337-467a-ae4f-18d92a7078c3.png">

### Che version

next (development version)

### Steps to reproduce

Try one of the badges [in this repo README](https://github.com/che-incubator/backstage-demo-quarkus2/blob/master/README.md) to start an ephemeral workspace on Che team dogfooding instance.

### Expected behavior

When the workspace starts no PVC is attached to the workspace.

### Runtime

other (please specify in additional context)

### Installation method

other (please specify in additional context)

### Environment

other (please specify in additional context)

### Additional context

I am testing in Che team dogfooding instance

### Release Notes Text

The fix allows to start workspaces in ephemeral mode, and therefore faster.

**Comment by l0rd** [comment]
cc @ibuziuk

**Comment by amisevsk** [comment]
For clarity, it appears that the issue is that the `attributes` field in the devfile is being dropped somewhere in the dashboard. I'm unable to add attributes to existing workspaces either.

**Comment by nickboldt** [comment]
this sounds like a temporary regression, fixed before anyone will encounter it in 3.0; not sure though, perhaps @l0rd has a different opinion on whether this merits inclusion in downstream N&N
